app/assets/javascripts/discourse/app/controllers/user-activity-bookmarks.js

```diff
@@ -19,6 +24,19 @@ export default Controller.extend({
 
   queryParams: ["q"],
 
+  init() {
+    this._super(...arguments);
+
+    scheduleOnce("afterRender", function() {
```

Comment:
scheduleOnce is useless here, because you pass it an anonymous function, so even if it was called multiple times (which is probably not) it's still considered a new function each time and running it again

Reply:
also it would be better in didInsertElement than init